browser/brave_stats/brave_stats_updater_params.cc

```diff
@@ -100,6 +101,57 @@ std::string BraveStatsUpdaterParams::GetProcessArchParam() const {
   }
 }
 
+std::string BraveStatsUpdaterParams::GetWalletEnabledParam() const {
+  base::Time wallet_last_unlocked =
+      profile_pref_service_->GetTime(kBraveWalletLastUnlockTime);
+  uint8_t usage_bitset = UsageBitstringFromTimestamp(wallet_last_unlocked);
```

Comment:
What stops this from reporting a daily active user twice if the browser is restarted?  Could you also add a test case for that, that a single user can't self report twice in a row on the same day?

Reply:
None of that logic changed in this PR. This just adds a param based off the timestamp you created. We already guard against that for sending the ping at all here https://github.com/brave/brave-core/blob/c9ba8ea1582145e11b26d849135094ffd8c2c257/browser/brave_stats/brave_stats_updater.cc#L288-L293

We can create that test case as a followup PR.